source/support.tex

```diff
@@ -583,6 +583,7 @@
 #define @\defnlibxname{cpp_lib_chrono}@                            201907L // also in \libheader{chrono}
 #define @\defnlibxname{cpp_lib_chrono_udls}@                       201304L // also in \libheader{chrono}
 #define @\defnlibxname{cpp_lib_clamp}@                             201603L // also in \libheader{algorithm}
+#define @\defnlibxname{cpp_lib_cmp_equal}@                         202002L // also in \libheader{utility}
```

Comment:
Per the motion, the feature-test macro should be `__cpp_lib_integer_comparison_functions`.

Reply:
done